### PR TITLE
Database connection

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

This PR allows for users to connect to the database through their environment variable. 

## Resolves Issue Number

#8

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout database-connection
dotnet restore
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Make sure to recompile code, as well as restore the JSON file as dependencies are changed. 
2. You must also make sure that your environment variable is set prior to compiling and running this PR.

## Impacted Files in Application
List general components of the application that this PR will affect:

* data/bangazonconnection.cs

### CODE REVIEW

**Comment by matthamil** [comment]
LGTM 🐑 💨

File: src/customer-service/Data/BangazonConnection.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        /**
+         * Purpose: Inserts new information into the datase
+         * Arguments:
+         *     query - database query
+         * Return:
+         *     n/a 
+         */
+        public void insert(string query)
+        {
+            SqliteConnection dbcon = new SqliteConnection(_path);
```
Comment (MattKraatz) [inline src/customer-service/Data/BangazonConnection.cs:28]: I think this needs that Data Source qualifier Data Source=
`SqliteConnection dbcon = new SqliteConnection(@"Data Source=" + _path);`

File: src/customer-service/Data/BangazonConnection.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+         * Arguments:
+         *     query - database query
+         *     Action<SqliteDataReader> handler - callback function to be executed during connection
+         * Return:
+         *     n/a 
+         */
+        public void execute(string query, Action<SqliteDataReader> handler)
+        {
+
+            SqliteConnection dbcon = new SqliteConnection(_path);
```
Comment (MattKraatz) [inline src/customer-service/Data/BangazonConnection.cs:51]: Same deal here with Data Source

**Review by MattKraatz — APPROVED** [review]
LGTM after quick change

File: src/customer-service/Data/BangazonConnection.cs (hunk 1 above)
Comment (matthamil) [reply src/customer-service/Data/BangazonConnection.cs:28]: Fixed in commit https://github.com/TeamCharles/customer-service/pull/9/commits/7abd15594deb8eb30e62a8a9d06350e4976c1cb3

File: src/customer-service/Data/BangazonConnection.cs (hunk 2 above)
Comment (matthamil) [reply src/customer-service/Data/BangazonConnection.cs:51]: See https://github.com/TeamCharles/customer-service/pull/9/commits/7abd15594deb8eb30e62a8a9d06350e4976c1cb3

**Review by Anulfo — COMMENTED** [review]
LGTM

**Comment by daynewright** [comment]
LGTM

**Pull request merged**